booking-engine-widget/integrations/google-triggers-reference.md

```diff
@@ -336,6 +341,24 @@ We are pushing all available variables into the eCommerce tracking for more adva
       adults: 2, // Count of adults in  reservation
       children: 0 // Count of children in reservation
     }
+    {
+      item_id: "ec34da8b-0932-493e-a742-baf1af194789", // Product id
+      item_name: "Product 1", // Product name
+      item_category: "Product",
+      item_category2: "Jan's Hotel", // Property name
+      item_category3: "Jan's Hotel City", // Property city
+      item_brand: "Jan's Hotel Chain", // Property chain
+      affiliation: "Jan's Hotel",
+      currency: "EUR",
+      price: 20, // Price of product
+      netPrice: 18.7, // Price of product without taxes
+      grossPrice: 20,  // Price of product with taxes
+      pricingMode: "Gross", // Pricing mode (Gross mainly Europe = tax is included in price, Net mainly US = tax non included in price)
+      quantity: 1, // Count of products
+      reservationGroupId: "2cf7e37b-b1d1-4970-8aaf-af0000f580f2",
+      reservationGroupName: "Name-29-8-8617", // Reservation Group name – can be mapped to export from mews Operations for each reservation
+      tax: 1.3, // Tax of product
```

Comment:
This is how view item list looks like:
![Screenshot 2024-03-22 at 14 33 13](https://github.com/MewsSystems/gitbook-booking-engine/assets/6526378/5771d236-53a8-46a3-95b8-15b79e667f63)

But I guess we rather want to make this product items match what we had before?